### PR TITLE
New/treeview navigation styling

### PR DESCRIPTION
**Pull request by yleaxeman**

### CODE REVIEW

**Review by proand — CHANGES_REQUESTED** [review]
Font size should be 19px, or is it just figma not beeing updated?

And the Material solution is a little bit more complex than the one in figma, but I guess thats a good thing.

File: src/blocks/_fhi-material-tree.block.scss [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+    .mat-tree-node {
+        align-items: unset;
+        display: flex;
+        min-height: $fhi-px * 44;
+    }
+}
+
+
+.fhi-material-tree__invisible {
```
[the remark this answers is not in the document]
Comment (yleaxeman) [reply src/blocks/_fhi-material-tree.block.scss]: I just discovered this, and nested it

**Comment by yleaxeman** [comment]
> 

The font size was forgotten.

**Review by proand — APPROVED** [review]
👍

**Pull request merged**